CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Rojo Changelog
 
 ## Unreleased Changes
+* Added support for `Enum` attributes
+* Significantly improved performance of `.rbxm` parsing
```

Comment:
It's likely that several common Rojo workloads beyond rbxm parsing see performance improvements with these changes. Maybe we can add a couple more benchmarks and report some rough figures for each scenario here?

Reply:
You're not incorrect, but I'm not sure what benchmarks are actually available. Not many projects that I have access to are big enough to benefit without also being heavily reliant upon `rbxm` parsing.

Reply:
Yeah, my idea was to be more user-centric about the nature of the performance differences, but we don't really have the time to create new benchmarks for this